include/RAJA/policy/openmp_target/reduce.hpp

```diff
@@ -205,6 +208,9 @@ struct TargetReduce
       }
     }
   }
+#ifdef __ibmxl__ // implicit declare target doesn't pick this up
```

Comment:
```suggestion
#ifdef __ibmxl__ // TODO: implicit declare target doesn't pick this up
```

Same suggestion as above.